mls-provider/tests/fixtures.rs

```diff
@@ -131,7 +131,7 @@ pub fn entropy() -> EntropySeed {
     openmls::prelude::Ciphersuite::MLS_128_DHKEMX25519_CHACHA20POLY1305_SHA256_Ed25519,
     Some(entropy())
 )]
-// TODO: The next ciphersuite isn't supported because of the lack of  ed448 (status unknown) crate
+// TODO: The next ciphersuite isn't supported because of the lack of ed448 (status unknown) crate. Tracking issue: WPB-9650
```

Comment:
I would suggest removing both this TODO and all ed448-related commented code below, since this feels
like a bigger thing, i.e. a feature, and doesn't look like a usual TODO.

The tracking issue for adding support for the new ciphersuite should be sufficient.